packages/web/.gitignore

```diff
@@ -35,3 +35,6 @@ yarn-error.log*
 
 # typescript
 *.tsbuildinfo
+
+# Sentry Auth Token
+.sentryclirc
```

Comment:
The auth token is not an ENV var?

Reply:
Yes it is, already  added it. This is to ignore the sentryirc file. This PR was a test though., I told Matt to push it from his PR 👍 . Closing this PR.